.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,28 @@
-# Pull Request Template
+The core Triton is a small number of people, and we receive many PRs (thank
+you!).  To help us review your code more quickly, **if you are a new
+contributor (less than 3 PRs merged) we ask that you fill out the following
+checklist and include it in your PR description.**
```

Comment:
Very clear instruction, thanks for the change. This will guide many contributors including myself.